Sources/LanguageServerProtocol/Types/LineIterator.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  LineIterator.swift
+//  LanguageServerProtocol
+//
+//  Created by Ryan Lovelett on 5/30/18.
+//
+
+private extension Character {
+
+    var isLineEnding: Bool {
+        return self == "\r\n" || self == "\r" || self == "\n"
+    }
+
+}
+
+struct Line {
```

Comment:
Add some documentation to `Line` to make sure we can remember what each of the fields is for.